src/Build/BuildCheck/Infrastructure/EditorConfig/EditorConfigParser.cs

```diff
@@ -12,6 +12,10 @@ namespace Microsoft.Build.Experimental.BuildCheck.Infrastructure.EditorConfig;
 
 internal sealed class EditorConfigParser
 {
+    // Define a static property to hold the editorConfigFilePath
+    private static ConcurrentBag<string> editorConfigFilePaths = new ConcurrentBag<string>();
```

Comment:
Editorconfig + binlog logic does not occur in VS right now, but when it eventually does wouldn't this create an ever growing collection that would be embedded? It should be cleaned up after embedding in binlog.